src/api/spec/services/webui_controller_service/user_checker_spec.rb

```diff
@@ -24,25 +24,6 @@
     end
   end
 
-  describe '#login_exists?' do
-    let(:request) { OpenStruct.new(env: { 'HTTP_X_USERNAME' => user_login }) }
```

Comment:
ok, you embedded the instance methods in `if/else` inside of the `call` and you don't need to test it anymore? Wouldn't be possible to test call return in different conditions with multiple `context`?

Reply:
I plan to rework the specs a bit. I realized this yesterday evening when seeing the coverage report.